dhis-2/dhis-api/src/main/java/org/hisp/dhis/commons/util/TextUtils.java

```diff
@@ -737,6 +737,10 @@ public static String cleanUrlPathOnly(@Nonnull String baseUrl, @Nonnull String p
    * @return a sanitized string.
    */
   public static String sanitize(Pattern pattern, String input, char replacement) {
+    if (StringUtils.isEmpty(input)) {
```

Comment:
```suggestion
    if (StringUtils.isBlank(input)) {
```

Reply:
Right. I am thinking this is a generic sanitizer method and there could be other usages of it.